index.js

```diff
@@ -38,17 +66,40 @@
  * And returns the first and last name concatenated together (i.e. 'Bob Evans').
  */
 
+function getFullName(person) {
+	var fullName = person.firstName + " " +person.lastName;	
+	console.log("Full Name: " + fullName);
+	return fullName;
+}
+
+var person = {
+	firstName : "Julia",
+	lastName : "Graves"
+};
 
 /**
  * Q6: Write a function called `reverseString` that has a String parameter and returns
  * the String reversed. Hint: look up `.split()`
  */
 
+function reverseString(string) {
+	var reversed = string.split("").reverse().join("");	
+	console.log(string + " reversed is " + reversed);
+	return reversed;
+}
 
 /**
  * Q7: Write a function called `maxArray` that has one Array of Numbers parameter and returns
  * the largest number in the Array. Use a `forEach` loop.
  */
+function maxArray(numbers) {
+	var largest = 0
+	for (var i = 0; i < numbers.length; i++) {
+    	 largest = numbers[i] > largest ? numbers[i] : largest;
+    }
+    console.log("the largest number in the array is " + largest);
+    return largest;
```

Comment:
A couple things here:

1. What if the numbers are all negative? Since you start with the largest at 0, 0 would get returned even if it isn't in the array. Perhaps a better starting point would be the first item in the array:

```
var largest = numbers[0];
```

2. This solution using a `for` loop is fine, but if we wanted to use a `forEach` loop like the question asked, it would look something like this:

```
numbers.forEach(function (number) {
    largest = number > largest ? number : largest;
});
```